numpy/lib/arraypad.py

```diff
@@ -207,23 +207,18 @@ def _get_linear_ramps(padded, axis, width_pair, end_value_pair):
     """
     edge_pair = _get_edges(padded, axis, width_pair)
 
-    left_ramp = np.linspace(
-        start=end_value_pair[0],
-        stop=edge_pair[0].squeeze(axis),  # Dimensions is replaced by linspace
-        num=width_pair[0],
+    left_ramp, right_ramp = (
+    np.linspace(
+        start=end_value,
+        stop=edge.squeeze(axis), # Dimension is replaced by linspace
+        num=width,
         endpoint=False,
         dtype=padded.dtype,
-        axis=axis,
-    )
-
-    right_ramp = np.linspace(
-        start=end_value_pair[1],
-        stop=edge_pair[1].squeeze(axis),  # Dimension is replaced by linspace
-        num=width_pair[1],
-        endpoint=False,
-        dtype=padded.dtype,
-        axis=axis,
+        axis=axis
```

Comment:
Can you indent all the lines between the opening `(` and this one, as I do in my comment?